pages/linux/vcgencmd.md

```diff
@@ -0,0 +1,28 @@
+# vcgencmd
+
+> Print system information for a Raspberry Pi.
+> More information: <https://www.raspberrypi.org/documentation/raspbian/applications/vcgencmd.md>.
+
+- List all available commands:
+
+`vcgencmd commands`
+
+- Print the current CPU temperature:
+
+`vcgencmd measure_temp`
+
+- Print the current voltage usage:
+
+`vcgencmd measure_volts`
+
+- Print whether the CPU has throttled:
+
+`vcgencmd get_throttled`
+
+- Print the bootloader config (only available on Raspberry Pi 4 models):
```

Comment:
I don't think there's a version 5 yet, but when there is, this page won't be outdated this way.
```suggestion
- Print the bootloader config (only available on Raspberry Pi 4 and above):
```

Reply:
We don't know whether it will have a EEPROM bootloader so I don't know about this one